libultraship/libultraship/BitConverter.h

```diff
@@ -0,0 +1,186 @@
+#pragma once
```

Comment:
Does this class need to be in LUS? Seems to me like the new full featured binary reader/writer does everything this was needed for.

Reply:
I'll take a closer look, I think it has the Endianness enum in it but I can probably just strip that out and put it somewhere else.

Reply:
On closer look, this is needed to handle switching Endianness, all of the functions in there are used and it makes sense to be a separate file because the BinaryWriter (which we'll probably want but is unused currently in libultraship) will also need it as well.

Reply:
If they're used in the new wrapper then it's fine. I thought you were using intrinsics in the new wrapper.

Reply:
Yeah I used this in the new wrapper so the places in existing code that set endianness values would still work the same way. It does appear that this class in and of itself is using intrinsics though.